.github/CONTRIBUTING.markdown

```diff
@@ -1,105 +1,121 @@
-  test will do. Please stay in the confines of the current test suite and use
-  [Shoulda](https://github.com/thoughtbot/shoulda/tree/master) and
-  [RSpec-Mocks](https://github.com/rspec/rspec-mocks).
-* If it's a brand new feature, make sure to create a new
-  [Cucumber](https://github.com/cucumber/cucumber/) feature and reuse steps
-  where appropriate. Also, whipping up some documentation in your fork's `site`
-  would be appreciated, and once merged it will be transferred over to the main
-  `site`, jekyllrb.com.
-* If your contribution changes any Jekyll behavior, make sure to update the
-  documentation. It lives in `site/_docs`. If the docs are missing information,
-  please feel free to add it in. Great docs make a great project!
-* Please follow the [GitHub Ruby Styleguide](https://github.com/styleguide/ruby)
-  when modifying Ruby code.
-* Please do your best to submit **small pull requests**. The easier the proposed
-  change is to review, the more likely it will be merged.
-* When submitting a pull request, please make judicious use of the pull request
-  body. A description of what changes were made, the motivations behind the
-  changes and [any tasks completed or left to complete](http://git.io/gfm-tasks)
-  will also speed up review time.
-
-Test Dependencies
------------------
-
-To run the test suite and build the gem you'll need to install Jekyll's
-dependencies. Simply run this command to get all setup:
+# Contributing to Jekyll
 
-    $ script/bootstrap
+Hi there! Interested in contributing to Jekyll? We'd love your help. Jekyll is an open source project, built one contribution at a time by users like you.
 
-Before you start, run the tests and make sure that they pass (to confirm your
-environment is configured properly):
+## Where to get help or report a problem
 
-    $ script/cibuild
+* If you have a question about using Jekyll, start a discussion on https://talk.jekyllrb.com.
+* If you think you've found a bug within a Jekyll plugin, open an issue in that plugin's repository.
+* If you think you've found a bug within Jekyll itself, [open an issue](https://github.com/jekyll/jekyll/issues/new)
 
-If you are only updating a file in `test/`, you can use the command:
+## Ways to contribute
 
-    $ script/test test/blah_test.rb
+Whether you're a developer, a designer, or just a Jekyll devotee, there are lots of ways to contribute. Here's a few ideas:
 
-If you are only updating a `.feature` file, you can use the command:
+* [Install Jekyll on your computer](https://jekyllrb.com/docs/installation/) and kick the tires. Does it work? Does it do what you'd expect? If not, [open an issue](https://github.com/jekyll/jekyll/issues/new) and let us know.
+* Comment on some of the project's [open issue](https://github.com/jekyll/jekyll/issues). Have you experienced the same problem? Know a work around? Do you have a suggestion for how the feature could be better?
```

Comment:
Could you make "issue" plural in the link? IMO, "project's open issues" reads better in the context of that sentence.